shared/AppInsightsCore/src/JavaScriptSDK/SenderPostManager.ts

```diff
@@ -379,8 +380,7 @@ export class SenderPostManager {
                 }
 
 
-                if (_sendCredentials && _isOneDs) {
-                    // for 1ds, Don't send credentials when URL is file://
+                if (_sendCredentials) {  // fetch request credentials
```

Comment:
this is also related to credentials. also not sure we should change it or not as this is for fetch request instead of xhr.